tests/remote/check_manual.tftest.hcl

```diff
@@ -179,7 +179,7 @@ run "resource_check_manual_update_unset_optional_fields" {
   variables {
     category         = run.from_rubric_category_get_category_id.first_category.id
     enable_on        = null
-    enabled          = !var.enabled
+    enabled          = null
```

Comment:
what does "null" mean for a check's enabled field?

Reply:
It's the same as excluding enabled from the config and does not actually unset the value. This actually could fit in it's own category